UPGRADE-API-1.11.md

```diff
@@ -63,6 +63,28 @@
           "description": "Sample description"
       } 
     ```
+   
+1. The new subresourceOperation has been added to `Sylius/Bundle/ApiBundle/Resources/config/api_resources/PaymentMethod.xml` file and now `api/v2/shop/payment/{id}/methods` endpoint uses his own subresource:
+   
+    ```diff
+           <subresourceOperations>
+              <subresourceOperation name="api_orders_payments_methods_get_subresource">
+    -             <attribute name="method">GET</attribute>
+                  <attribute name="normalization_context">
+                      <attribute name="groups">
+                          <attribute>shop:payment_method:read</attribute>
+                      </attribute>
+                  </attribute>
+              </subresourceOperation>
+    +         <subresourceOperation name="api_payments_methods_get_subresource">
+    +             <attribute name="normalization_context">
+    +                 <attribute name="groups">
+    +                     <attribute>shop:payment_method:read</attribute>
+    +                 </attribute>
+    +             </attribute>
+    +         </subresourceOperation>
+          </subresourceOperations>
+    ```
```

Comment:
```suggestion
1. The `api/v2/shop/payment/{id}/methods` endpoint has now `shop:payment_method:read` serialization group assigned. Therefore its body will look like this by default:
    ```
    {
        "@context": "\/api\/v2\/contexts\/PaymentMethod",
        "@id": "\/api\/v2\/shop\/orders\/nAWw2jewpA\/payments\/@integer@\/methods",
        "@id": "\/api\/v2\/shop\/payments\/@integer@\/methods",
        "@type": "hydra:Collection",
        "hydra:member": [
            {
                "@id": "\/api\/v2\/shop\/payment-methods\/CASH_ON_DELIVERY",
                "@type": "PaymentMethod",
                "id": 1,
                "code": "CASH_ON_DELIVERY",
                "position": 0,
                "name": "Cash on delivery",
                "description": "Description",
                "instructions": null
            }
        ],
        "hydra:totalItems": 1
    }
    ```
```

Perhaps, we cou